app/models/operatingsystem.rb

```diff
@@ -22,7 +22,7 @@ class Operatingsystem < ApplicationRecord
   accepts_nested_attributes_for :os_default_templates, :allow_destroy => true,
     :reject_if => :reject_empty_provisioning_template
 
-  validates :major, numericality: true, presence: { message: N_("Operating System version is required") }
+  validates :major, numericality: {:greater_than_or_equal_to => 0, :message => N_("Major version should be greater than or equal to 0") }, presence: { message: N_("Operating System version is required") }
```

Comment:
```suggestion
  validates :major, numericality: {:greater_than_or_equal_to => 0, :message => N_("Major version of the operating system must be greater than or equal to 0") }, presence: { message: N_("Operating System version is required") }
```
I'm not sure what the context is for this text, but I'd include the words "operating system".

Reply:
addressed